Automattic/woocommerce-connect-client
# Support printing multiple labels at once

Fixes #500 

To test this, you will need also the `try/pdf-manipulation` server-side branch.

This PR:
* Makes the `PREVIEW` box in the Labels UI a bit more useful. The merchant can select a paper size and preview how the PDF he'll have to print looks.
* Adds a `Paper Size` dropdown to the Labels UI. The selected value is remembered forever, so the merchant will only have to change it once (and if he buys a new printer).
* Adds the same `Paper Size` dropdown to the `Reprint` dialog.
* The rest of the changes are "under the hood", on the way the PDF is actually generated (a GET request to the merchant host that is transformed into a POST request to the server).

How to test:
* Create an order with multiple items.
* Experiment with the `Paper size` dropdown in the Labels UI and how that affects the Preview.
* Buy labels for all those items.
* Check that the PDF to print has the expected page size and number of labels.
* Try to reprint a newly purchased label.

@nabsul @robobot3000 @jeffstieler @allendav @kellychoffman
